src/types.ts

```diff
@@ -0,0 +1,7 @@
+import { RequiredOptions } from 'prettier';
+
+export interface PrettierOptions extends RequiredOptions {
+    importOrder: string[];
+    importOrderSeparation: boolean;
+    experimentalBabelParserPluginsList: any[];
```

Comment:
is not `experimentalBabelParserPluginsList
`  always a list of string ?

Reply:
I added some type safety for `experimentalBabelParserPluginsList` now and it will be a `ParserPlugin[]` which makes more sense.